### DISCUSSION TITLE
The `bg-brand-500` class does not exist.

### DISCUSSION THREAD

**Discussion by BrekiTomasson** [tailwindlabs/tailwindcss#5680] [Help] [answered]
I've looked at other discussions here that are related to similar error messages, but none of them (at least none that I've found so far) seem to apply to my problem. I'm running `"tailwindcss": "^3.0.0-alpha.1"`, and have the following in my tailwind config file:

```
[...]
  theme: {
    extend: {
      colors: {
        brand: {
          50: 'rgb(244,250,251)',
          100: 'rgb(231,246,248)',
          200: 'rgb(201,236,240)',
          300: 'rgb(160,219,232)',
          400: 'rgb(95,188,220)',
          500: 'rgb(47,152,203)',
          600: 'rgb(34,116,175)',
          700: 'rgb(29,95,145)',
          800: 'rgb(31,70,101)',
          900: 'rgb(26,56,79)',
        },
      },
[...]
```

This used to work by adding 'brand' as a separate color to the palette and generating the relevant classes where required, allowing me to `<div class="bg-brand-300">` etc..

However, using `@apply bg-brand-500` anywhere in my SCSS or Vue `<style>` tags breaks the build process with this error:

`(1:12413) /var/www/resources/sass/App.scss The `bg-brand-200` class does not exist. If `bg-brand-200` is a custom class, make sure it is defined within a `@layer` directive.`

Are "extended" colors treated differently somehow? It seems to work fine if I just use the default colors.

**Comment by adamwathan** [comment]
Hmm should definitely work! Any chance you can put together a demo I can play with?

**Comment by BrekiTomasson** [comment, marked as answer]
I managed to figure it out, there was an incompatibility with a TailwindCSS plugin from back in the 1.x era. Once I disabled it, everything worked as intended.
